shacl12-vocabularies/shnex.ttl

```diff
@@ -400,6 +400,32 @@ shnex:minus
 .
 
 
+# Nodes Where Expressions -----------------------------------------------------
```

Comment:
If every `Nodes Where` is case-sentively replaced with `Nodes Matching`, I think @afs will be satisfied, and general comprehension improved.

Reply:
Yes